src/Controllers/IntegrationController.php

```
@@ -434,9 +433,9 @@ public function jiraSearchUsers(): void
                 'email'       => $u['emailAddress'] ?? '',
                 'avatar'      => $u['avatarUrls']['24x24'] ?? '',
             ], $users);
-            echo json_encode(['users' => array_values($mapped)]);
+            Response::sendJson(['users' => array_values($mapped)]);
         } catch (\Throwable $e) {
-            echo json_encode(['users' => [], 'error' => $e->getMessage()]);
+            Response::sendJson(['users' => [], 'error' => $e->getMessage()]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\StratFlow\Core\Response' in method 'jiraSearchUsers'. (undefined)

(StaticAccess)